src/models/activityReport.js

```diff
@@ -142,9 +142,6 @@ export default (sequelize, DataTypes) => {
     requester: {
       type: DataTypes.STRING,
     },
-    programTypes: {
-      type: DataTypes.ARRAY(DataTypes.STRING),
-    },
```

Comment:
Is there an issue with this being removed here without changing the DB?

Reply:
I don't think so as all BE tests pass and I can create and retrieve a report on the site without issue.